src/components/common/TokenImg.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { TokenImg, Props } from './TokenImg'
+import { WETH_ADDRESS_MAINNET, WETH_ADDRESS_RINKEBY } from 'const'
+import { Frame } from './Frame'
+import styled from 'styled-components'
+
+const ADDRESS_GNO = '0x6810e776880C02933D47DB1b9fc05908e5386b96'
```

Comment:
Didn't we have a GNO address somewhere in constants? Or is this xDAI GNO?

Reply:
mocked, with no address. But could refactor this into the mocks too

Reply:
done